doc/README.rst

```diff
@@ -118,8 +125,13 @@ there:
 
    cd ~/zephyr
    source zephyr-env.sh
+
+   # To generate HTML output
    make htmldocs
 
+   # To generate PDF output
+   make pdfdocs
```

Comment:
This won't work from the root of Zephyr as @nashif mentions. you need a new target in the top-level `Makefile`